completions/bzip2

```diff
@@ -37,7 +37,8 @@ _comp_cmd_bzip2()
     _comp_compgen_tilde && return
 
     compopt -o filenames
-    COMPREPLY=($(compgen -f -X "$xspec" -- "$cur") $(compgen -d -- "$cur"))
+    _comp_compgen -- -f -X "$xspec"
+    _comp_compgen -a -- -d
```

Comment:
Maybe while at it and in similar cases throughout this PR?
```suggestion
    _comp_compgen -- -f -X "$xspec" -o plusdirs
```

Reply:
Ah, you are right. I'll later look them up in the codebase.

Reply:
Applied in 4358f1e5a1e3f9a3a292b6d2b7a7f27f1611907e